### ISSUE TITLE
[FEATURE REQUEST] citrixsdx_adminprofile

### ISSUE THREAD

**Issue by HS1542** [citrix/terraform-provider-citrixadc#997]
**First of all, sorry to post this here, but it seems like you haven't opened up for registering issues for the citrixsdx provider:
https://github.com/citrix/terraform-provider-citrixsdx**

# Feature Request
When you provision a VPX, you want to be able to provide an admin profile with the initial password for nsroot and SNMP settings.
It doesn't seem to me like the provider will let you create an admin profile, but requires you to reference an already existing admin profile.
If you don't specify an admin profile it will default to "ns_nsroot_profile"
It is not documented and I cannot seem to understand what profile_username or profile_password would do, as these seems overrun by settings of the chosen admin profile?
The username and password parameters will create a different instance administrator with the limited "superuser" command policy and hence not full sysadmin.

**To summarize: I would very much like to see that the citrixsdx provider could give us the possibility to create an admin profile in advance or directly within the citrixsdx_provision_vpx resource.**
